src/components/backend-ai-import-view.ts

```diff
@@ -6,7 +6,7 @@
 import {get as _text, translate as _t} from 'lit-translate';
 import {css, CSSResultGroup, html} from 'lit';
 import {customElement, property} from 'lit/decorators.js';
-
+import {unsafeHTML} from 'lit/directives/unsafe-html.js';
```

Comment:
I don't see any usage of the `unsafeHTML` in this PR. If it is not used,please remove it.